src/pyuvsim/data/test_config/28m_triangle_10time_10chan_nodiameter.yaml

```diff
@@ -1,12 +1,13 @@
 beam_paths:
-  0:
+  0: !UVBeam
     filename: HERA_NicCST.beamfits
-  1:
-    type: uniform
-  2:
-    type: gaussian
+    path_variable: pyuvsim.data.DATA_PATH
```

Comment:
I see how you're using the `path_variable` here (cf my question on `pyuvdata`). It's possible we could use entry-points to define "plugins" to pyuvdata that add to a global "BEAM_PATHS" list, which could be searched through for filenames, in which case we wouldn't need this variable at all. Lemme know if you're interested in that.

Reply:
Interesting. I really think this is primarily needed for testing, I don't expect it to really be used by users much at all, but I needed to make it work for the pyuvsim tests (I wasn't planning to advertise it in the docs). If you think the plugin structure is better we can explore it, but since this is really just for testing I don't know that we need to put a lot of effort into it.

Reply:
Yeah, OK let's keep as-is for now. I think it's clear enough.